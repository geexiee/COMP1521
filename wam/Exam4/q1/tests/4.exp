Loaded: /home/class/lib/exceptions.s
s1 = T
s1 = T
s2 = T
nv = 0
