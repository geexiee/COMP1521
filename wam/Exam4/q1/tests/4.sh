exe tests/s4.s
