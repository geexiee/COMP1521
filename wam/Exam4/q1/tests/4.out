Loaded: /home/cs1521/lib/spim/exceptions.s
s1 = T
s1 = T
s2 = TXX
nv = 0
